CMakeLists.txt

```diff
@@ -59,8 +59,8 @@ list(APPEND CMAKE_MODULE_PATH ${LLVM_DIR})
 
 include(AddLLVM)
 
-set(LLVM_TOOLS_BINARY ${LLVM_INSTALLATION_DIR}/bin)
-set(LLVM_LIT_TOOLS_DIR$ ${LLVM_INSTALLATION_DIR}/bin)
+set(LLVM_LIT_TOOLS_DIR ${LLVM_INSTALATION_DIR} CACHE STRING "Command used to spawn lit")
```

Comment:
One of these STRING values is wrong (both variables can't do the same thing surely?) and I assume it is this one.

Reply:
I've got this from here:

Reply:
Done!